src/test/java/CarTest.java

```diff
@@ -0,0 +1,69 @@
+import domain.Car;
+import domain.Cars;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class CarTest {
+
+    @BeforeEach
+    void setUp() {
+        Cars.cars = null;
+        Cars.carNames = null;
+    }
+
+    @DisplayName("자동차 이름 분리 테스트")
+    @Test
+    void testSplitCarName(){
+        String input = "car1,car2,car3";
+
+        Cars.splitCarName(input);
+
+        assertEquals(3, Cars.carNames.length);
+        assertArrayEquals(new String[]{"car1","car2","car3"},Cars.carNames);
```

Comment:
이번 주차의 학습 내용에 포함되는 `JUnit`과 `AssertJ`도 활용해보면 좋을 것 같아요.
JUnit이 아닌, AssertJ에서 제공하는 검증 메서드를 사용해보면 어떨까요?

```java
import org.assertj.core.api.Assertions;
```
```suggestion
        Assertions.assertThat(Cars.carNames.length).isEqualTo(3);
        Assertions.assertThat(Cars.carNames).contains("car1","car2","car3");
```

ref)
- Assertj doc: https://assertj.github.io/doc/#assertj-core-assertions-guide
- assertj 사용 참고자료: https://bibi6666667.tistory.com/231

Reply:
넵 수정했습니다.